BUILD.gn

```diff
@@ -199,6 +180,47 @@ component("node_lib") {
     defines += [ "NODE_RELEASE_URLBASE=\"$node_release_urlbase\"" ]
   }
 
+  if (node_use_openssl) {
+    defines += [
+      "NODE_OPENSSL_SYSTEM_CERT_PATH=\"$node_openssl_system_ca_path\"",
+      "EVP_CTRL_CCM_SET_TAG=EVP_CTRL_GCM_SET_TAG",
+    ]
+  }
+}
+
+component("node_lib") {
+  deps = [
+    ":node_js2c",
+    "deps/cares",
+    "deps/http_parser",
+    "deps/llhttp",
+    "deps/nghttp2",
+    "deps/zlib",
+    "//v8:v8_libplatform",
+  ]
+  public_deps = [
+    "deps/uv",
+    "//electron:atom_js2c",
+    "//v8",
+  ]
+  configs += [ ":node_internal_config" ]
+  public_configs = [
+    ":node_lib_config",
+    ":node_features",
```

Comment:
You can have "node_lib_config" inherit "node_features" by adding
`configs += ":node_features"`
to the former